app/src/main/java/org/simple/clinic/bloodsugar/history/adapter/BloodSugarHistoryListItem.kt

```diff
@@ -96,6 +116,18 @@ sealed class BloodSugarHistoryListItem : ItemAdapter.Item<Event> {
       } else {
         holder.bloodSugarIconImageView.setImageResource(R.drawable.ic_blood_sugar_outline)
       }
+
+      // TODO (SM): Removing edit functionality to disallow user from opening update sheet,
+      //  will be available once remove blood sugar functionality is implemented
+
+      //      if (isBloodSugarEditable) {
+      //        holder.itemView.setOnClickListener { subject.onNext(BloodSugarHistoryItemClicked(measurement)) }
+      //      } else {
+      //        holder.itemView.setOnClickListener(null)
+      //      }
+      //      holder.itemView.isClickable = isBloodSugarEditable
+      //      holder.itemView.isFocusable = isBloodSugarEditable
+      //      holder.editButton.visibleOrGone(isBloodSugarEditable)
```

Comment:
Instead of commenting code, can you think of maybe adding a flag somewhere in the `BloodSugarModule` maybe?